ManagementFrontendDockerfile

```diff
@@ -3,6 +3,8 @@ FROM frolvlad/alpine-gxx
 COPY ./ /clipper
 
 RUN apk add --no-cache git bash make boost-dev cmake libev-dev hiredis-dev zeromq-dev \
+    && cd /clipper/src/libs/spdlog \
+    && git apply ../patches/make_spdlog_compile_linux.patch \
```

Comment:
I didn't want to fork spdlog just for this tiny change for the Docker builds, so I just created a patch that we can apply.